ProcessLib/ThermoMechanics/ThermoMechanicsProcessData.h

```diff
@@ -76,6 +78,7 @@ struct ThermoMechanicsProcessData
     ParameterLib::Parameter<double> const& reference_solid_density;
     ParameterLib::Parameter<double> const& linear_thermal_expansion_coefficient;
     ParameterLib::Parameter<double> const& specific_heat_capacity;
+    ParameterLib::Parameter<double> const* const nonequilibrium_stress;
```

Comment:
Move this line one lower, after the `thermal_conductivity`. This causes warnings otherwise: https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-2544/3/gcc/

Reply:
OK, changed.